### DISCUSSION TITLE
Randomly generating resolver functions for fields

### DISCUSSION THREAD

**Discussion by theprobugmaker** [99designs/gqlgen#1866] [Q&A]
### What happened?

I realized that sometimes when I run `go generate ./...` the `*.resolvers.go` generated file has a few functions for fields.

For example:

```graphql
type Todo {
  id: ID!
}

type Query {
  todos: [Todo!]!
}
```

The generated `*.resolvers.go` file generates a function like (also it generates the `todos` query function normally):

```go
func (r *todoResolver) ID(ctx context.Context, obj *graph.Todo) (int, error) {
  // ...
}
```

### What did you expect?

Not to generate that?! Is that supposed to happen? It wasn't generating before and out of a sudden started to generate that.

### versions
 - `gqlgen version`? 0.16.0
 - `go version`? 1.16

**Comment by frederikhors** [comment]
Please post a reproduction as a Github repository.
